tests/requirements.txt

```diff
@@ -1,9 +1,9 @@
-flexmock
+flexmock>=0.10.3
 ordereddict
 responses>=0.9.0
 pypng
-pytest==4.0.*
-pytest-cov==2.6.*
+pytest>=4.1.0
```

Comment:
Should we straight-up require `pytest>=5.0.0`? I believe pip installs the latest version anyway. 4.1.0 is the version which introduces an API change that breaks flexmock (which is why we need `flexmock>=0.10.3`)

Reply:
That would require having separate requirements for py2, as pytest>=5.0.0 is not available there.